### PR TITLE
4/20 New news: Listen to Customers by Asking the Right Questions

### PR DESCRIPTION
**Pull request by ToniBonittoGSA**
Four Questions to Help Discover What Customers Want and Need

---

**Preview:** 
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/TWIDEA-8/2020/04/20/listen-customers-by-asking-right-questions/

### CODE REVIEW

File: content/news/2020/04/2020-04-20-listen-customers-by-asking-right-questions.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+Use common profiles or [personas](https://methods.18f.gov/decide/personas/) to better understand customer needs, and improve your site to meet those needs. For example, [Code.gov uses four personas](https://github.com/GSA/code-gov/tree/master/docs/UX) to inform their content and site structure. 
+
+Also, follow these [tips for starting your customer experience journey](https://www.performance.gov/tips-for-starting-your-customer-experience-journey/). These are low-cost, low-lift actions you can take to develop a customer profile. 
+
+### Q2. What tools can I use to gather feedback? 
+
+The good news is that there are several tools available to gather feedback. Here are a few tools to help you get started: 
+
+- **[Digital Analytics Program (DAP)](https://digital.gov/services/dap/)**  - An analytics tool for measuring digital services in the federal government. It also powers [analytics.USA.gov](http://analytics.USA.gov), which offers a quick window into your website. (Note, [M-17-06 requires DAP](https://policy.cio.gov/web-policy/analytics) on all federal executive branch public-facing websites.)
```
Comment (afeijoo) [inline content/news/2020/04/2020-04-20-listen-customers-by-asking-right-questions.md:74]: Update DAP URL from https://digital.gov/services/dap/ to https://digital.gov/guides/dap/.

File: content/news/2020/04/2020-04-20-listen-customers-by-asking-right-questions.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+# 1 -- visible
+weight: 1
+
+# Make it better ♥
+
+---
+
+{{< card-quote text="If you do not know how to ask the right question, you discover nothing." cite="W. Edwards Deming" >}}
+
+{{< img-right src="this-weeks-idea-icon-wk1" >}}
```
Comment (afeijoo) [inline content/news/2020/04/2020-04-20-listen-customers-by-asking-right-questions.md:42]: This image is broken.

Comment (ToniBonittoGSA) [reply content/news/2020/04/2020-04-20-listen-customers-by-asking-right-questions.md:42]: thx, will check

File: content/news/2020/04/2020-04-20-listen-customers-by-asking-right-questions.md (hunk 1 above)
Comment (ToniBonittoGSA) [reply content/news/2020/04/2020-04-20-listen-customers-by-asking-right-questions.md:74]: I did that on purpose; it's the real URL.. eventually, we will get back to a simple, single page under `/services/` which will be sep. from the group of pages in the `/guides/` directory.

**Pull request merged**